### PR TITLE
b2b - updates after review and re-run

### PR DESCRIPTION
**Pull request by Katerynaa-Lavryk**
b2b - updates after review and re-run

### CODE REVIEW

File: resources/environments/environments_api_b2b.json [hunk 1]
```diff
@@ -534,13 +550,13 @@
...
-                "description": "Get a 5% ount on all white products with voucher code",
-                "voucher_code": "sprykerka6b"
+                "description": "Get a 5% discount on all white products with voucher code",
+                "voucher_code": "sprykerca9j"
             },
             "id_4":{
                 "name": "10% off Safescan",
                 "description": "Get a 10% discount on all Safescan brand products with voucher code",
-                "voucher_code": "safescanye4ru9nu9b"
+                "voucher_code": "safescanqu1pu4ga1z"
```
Comment (OKrivtsova) [inline resources/environments/environments_api_b2b.json:559]: I would assume we need to get voucher codes from DB - they will be different on all envs and after redeployed

File: tests/api/b2b/glue/customer_endpoints/customer_restore_password/negative.robot [hunk 2]
```diff
@@ -22,10 +22,11 @@ Restore_password_with_empty_type
     And Response header parameter should be:    Content-Type    ${default_header_content_type}
 
 Restore_password_with_incorrect_type
-    I send a PATCH request:    /customer-restore-password/${yves_user.reference}   {"data":{"type":"customer-restore","attributes":{"restorePasswordKey":"5ec608df9c0dd57c3dd08b540d4a68da","password":"${yves_user.password}","confirmPassword":"${yves_user.password}"}}}
-    And Response status code should be:    400
-    And Response reason should be:    Bad Request
-    And Response should return error message:    Invalid type.
+    I send a PATCH request:    /customer-restore-password/${yves_user.reference}   {"data":{"type":"customer-restore-password","attributes":{"restorePasswordKey":"5ec608df9c0dd57c3dd08b540d4a68da","password":"${yves_user.password}","confirmPassword":"${yves_user.password}"}}}
```
Comment (OKrivtsova) [inline tests/api/b2b/glue/customer_endpoints/customer_restore_password/negative.robot:25]: you changed this test to test for a different thing (initially it tested for wrong type, now for the wrong key)
please rename the test

File: resources/environments/environments_api_b2b.json (hunk 1 above)
Comment (Katerynaa-Lavryk) [reply resources/environments/environments_api_b2b.json:559]: Added step to get voucher code from db

Comment (AntonShopin) [reply resources/environments/environments_api_b2b.json:559]: Please extend the steps when you get the voucher code from the DB. 
First - get the voucher_pool from the DB using discount ID
Then - get the code using this voucher_pool
Also, please revome voucher codes from the env file

Comment (Katerynaa-Lavryk) [reply resources/environments/environments_api_b2b.json]: Updated

**Review by AntonShopin — APPROVED** [review]

**Pull request merged**